docs/core/deploying/index.md

```diff
@@ -1,95 +1,104 @@
 ---
-title: .NET Core Application Deployment
-description: Learn about the ways to deploy a .NET Core application.
-ms.date: 12/03/2018
+title: Application publishing
+description: Learn about the ways to publish a .NET Core application. You can publish either a self-contained app or a runtime-dependent app.
+ms.date: 01/31/2020
 ---
-# .NET Core application deployment
+# .NET Core application publishing overview
 
-You can create three types of deployments for .NET Core applications:
+Applications you create with .NET Core can be published in two different ways. Each way affects how a user installs and runs your app. Publishing your app self-contained includes the .NET Core runtime and libraries along with all app dependencies and produces a platform-specific executable. Publishing your app runtime-dependent only contains your app with it's dependencies, and produces a cross-platform binary. Starting with .NET Core SDK 3.0, publishing an app runtime-dependent also produces a platform-specific executable.
```

Comment:
```suggestion
Applications you create with .NET Core can be published in two different ways. Publishing your app self-contained produces *.dll* files that contain your app and its dependencies, the .NET Core runtime and libraries, and a platform-specific executable. Publishing your app runtime-dependent produces a cross-platform *.dll* file that contains your app and its dependencies. Starting with .NET Core SDK 3.0, publishing an app runtime-dependent also produces a platform-specific executable.
```

Could also briefly summarize the difference of needing or not needing the runtime to be installed on the target machine.

If you prefer not to accept the suggestion, note that "it's" should still be corrected to "its".

Reply:
Whoops, I totally swapped that it's... 🐕 

I like it. How about this adjustment though? I feel like it's important to state that this really affects how a user runs the app. Do they or don't they need to have the runtime installed. (further detailed in the article sections)

```suggestion
Applications you create with .NET Core can be published in two different ways, each way affects how a user runs your app. Publishing your app self-contained produces a *.dll* file containing your app and includes: its dependencies, the .NET Core runtime and libraries, and a platform-specific executable. Publishing your app runtime-dependent produces a cross-platform *.dll* file that contains your app and its dependencies. Starting with .NET Core SDK 3.0, publishing an app runtime-dependent also produces a platform-specific executable for your current platform.
```

Reply:
Looks good, but I would make the first two sentences separate sentences or link them in some other way than just a comma.